templates/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta name="google-site-verification" content="VBSIFp1OyI1gpnRTREBEjNBj_Wat50wzwzVTrG1YuHY" />
+    <meta name="google-site-verification" content="7nOT58tx-TbWUt6dokG5Yj1Chca3j5-qX5qQ4dTGVew" />
```

Comment:
@dnuwa this should be raised in another PR.

Reply:
Noted